rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -1274,15 +1276,22 @@ void low_amount_release_request_rejected_before_lovell_value_in_satoshis() throw
 
         Coin belowPegoutMinimumValue = BRIDGE_CONSTANTS.getMinimumPegoutTxValue().minus(Coin.SATOSHI);
         co.rsk.core.Coin pegoutRequestValue = co.rsk.core.Coin.fromBitcoin(belowPegoutMinimumValue);
+        // Add some extra weis to the value, but less than 1 satoshi.
+        // To ensure that the pegout value is rounded down to fit in satoshis.
+        co.rsk.core.Coin oneSatoshiInWeis = co.rsk.core.Coin.fromBitcoin(Coin.SATOSHI);
+        co.rsk.core.Coin extraWeis = oneSatoshiInWeis.subtract(co.rsk.core.Coin.valueOf(1));
+        pegoutRequestValue = pegoutRequestValue.add(extraWeis);
 
         bridgeSupport.releaseBtc(buildReleaseRskTx(pegoutRequestValue));
 
         RskAddress senderAddress = new RskAddress(SENDER.getAddress());
 
+        // Pre lovell, the refund should be rounded down to the nearest satoshi.
```

Comment:
do you think this comment is necessary even having the other one [here](https://github.com/rsksmart/rskj/pull/2798/files#diff-e76588191a0bacc0270da08e9e8242cb6eb1c26b051a9e76ad7bb4e60376fafcR1264)?

Reply:
I actually do yes. I think we tend to focus on what's right in front of us